docker-bake.hcl

```diff
@@ -6,6 +6,14 @@ variable "REPOSITORY" {
   default = "oplabs-tools-artifacts/images"
 }
 
+# Proofs tools variables
+variable "PROOFS_CHALLENGER_VERSION" {
+  default = "b46bffed42db3442d7484f089278d59f51503049"
+}
```

Comment:
nit: I'd suggest just keeping this in proofs-tools since it's only referenced once and is potentially confusing since most targets will use the current source for challenger and it's only proofs-tools that uses a fixed challenger version.